numba_dpex/core/datamodel/models.py

```diff
@@ -106,3 +93,29 @@ def _init_data_model_manager():
 # Register the DpctlSyclQueue type with Numba's OpaqueModel
 register_model(DpctlSyclQueue)(OpaqueModel)
 dpex_data_model_manager.register(DpctlSyclQueue, OpaqueModel)
+
+
+@register_model(SyclQueueType)
+class SyclQueueModel(models.StructModel):
+    def __init__(self, dmm, fe_type):
+        members = [
+            ("backend", types.PyObject),
+            ("driver_version", types.StringLiteral),
```

Comment:
There is literally no point in add literals to model. If you need literals you should add it to type instead.